api/client/fip.go

```diff
@@ -35,20 +35,37 @@ func (cli *DockerCli) CmdFip(args ...string) error {
 // Usage: docker fip create [OPTIONS] COUNT
 func (cli *DockerCli) CmdFipAllocate(args ...string) error {
 	cmd := Cli.Subcmd("fip allocate", []string{"COUNT"}, "Creates some new floating IPs by the user", false)
+	flNew := cmd.Bool([]string{"f", "-force"}, false, "Force to allocate new floating IP")
```

Comment:
This is not "force allocate",  

1. **not allocate if have available** should not be enabled by default because it **changed** the semantics of this
2. name the option as `--pick-if-have-available`
3. or create a new command such as `get` or `fetch` or sth. else.